cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -236,7 +235,7 @@ function formatPlanForCheckout(plan, currencyCode) {
     }
     if (plan.installments_count >= 5 && planIsActivated(PaymentMgr.getPaymentMethod(ALMA_CREDIT_ID), plan)) {
         formatPlan = {
-            in_page: isAvailableForInpage(plan.installments_count, plan.deferred_days) && almaConfigHelper.isInpageActivated(),
+            in_page: isAvailableForInpage(plan.installments_count) && almaConfigHelper.isInpageActivated(),
             selector: getSelectorNameFromPlan(plan),
             installments_count: plan.installments_count,
             deferred_days: plan.deferred_days,
```

Comment:
not plan.deferred_months ?

Reply:
The feature deferred months is not available on SFCC